
syntax = "proto3";

package nyetflix;

service NyetFlixRPC {

	rpc SayHello (HelloRequest) returns (HelloReply) {}

	rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
	string name = 1;
}

message HelloReply {
	string message = 1;
}
